packages/core/vitest-integration.setup.ts

```diff
@@ -1,81 +1,68 @@
 import { Database, createKysely, createMigrator } from "@dotkomonline/db"
-import { Environment } from "@dotkomonline/env"
+import { createEnvironment, Environment } from "@dotkomonline/env"
 import { PostgreSqlContainer, type StartedPostgreSqlContainer } from "@testcontainers/postgresql"
 import { Kysely, sql } from "kysely"
-import { beforeAll } from "vitest"
+import { beforeAll, afterAll } from "vitest"
 
-// Configuration settings for the database container
-const testDBConfig = {
-  password: "local",
-  username: "local",
-  database: "main",
-  imageName: "public.ecr.aws/z5h0l8j6/dotkom/pgx-ulid:0.1.3",
-}
-
-// Used globally for all tests
 let container: StartedPostgreSqlContainer
-
-async function setupDatabaseContainer() {
-  const container = await new PostgreSqlContainer(testDBConfig.imageName)
-    .withExposedPorts(5432)
-    .withUsername(testDBConfig.username)
-    .withPassword(testDBConfig.password)
-    .withDatabase(testDBConfig.database)
-    .withReuse()
-    .start()
-
-  process.env.DATABASE_URL = container.getConnectionUri()
-  return container
-}
+let host: Kysely<Database>
 
 async function runMigrations(env: Environment, dbName: string) {
-  const db = getTestDb(env, dbName)
+  const db = createKyselyForDatabase(env, dbName)
   const migrator = createMigrator(db, new URL("node_modules/@dotkomonline/db/src/migrations", import.meta.url))
   await migrator.migrateToLatest().catch(console.warn)
   await db.destroy()
 }
 
-function buildDbUrl(dbName: string): string {
-  return `postgres://${testDBConfig.username}:${
-    testDBConfig.password
-  }@localhost:${container.getFirstMappedPort()}/${dbName}`
-}
-
-export function getTestDb(env: Environment, dbName: string): Kysely<Database> {
-  const url = buildDbUrl(dbName)
+function createKyselyForDatabase(env: Environment, dbName: string): Kysely<Database> {
   return createKysely({
     ...env,
-    DATABASE_URL: url,
+    DATABASE_URL: `postres://local:local@localhost:${container.getFirstMappedPort()}/${dbName}`,
```

Comment:
```suggestion
    DATABASE_URL: `postres://local:local@${container.getHost()}:${container.getFirstMappedPort()}/${dbName}`,
```

https://java.testcontainers.org/features/networking/
> Getting the container host
> When running with a local Docker daemon, exposed ports will usually be reachable on localhost. However, in some CI environments they may instead be reachable on a different host.

> As such, Testcontainers provides a convenience method to obtain an address on which the container should be reachable from the host machine.